console/src/telemetry/filters.ts

```diff
@@ -14,57 +14,53 @@ const DATA_PATH = '/data';
 const API_EXPLORER_PATH = '/api-explorer';
 const REMOTE_SCHEMAS_PATH = '/remote-schemas';
 const EVENTS_PATH = '/events';
+const ACTIONS_PATH = '/actions';
 
 const dataHandler = (path: string) => {
-  return (
-    DATA_PATH +
-    path
-      .replace(/\/schema\/([^/]*)(\/)?/, '/schema/SCHEMA_NAME$2')
-      .replace(
-        /(\/schema\/.*)\/tables\/([^/]*)(\/.*)?/,
-        '$1/tables/TABLE_NAME$3'
-      )
-      .replace(/(\/schema\/.*)\/views\/([^/]*)(\/.*)?/, '$1/views/VIEW_NAME$3')
-      .replace(
-        /(\/schema\/.*)\/functions\/([^/]*)(\/.*)?/,
-        '$1/functions/FUNCTION_NAME$3'
-      )
-  );
+  return path
+    .replace(/(\/schema\/)[^/]*(\/)?/, '$1SCHEMA_NAME$2')
+    .replace(/(\/schema\/.*\/tables\/)[^/]*(\/.*)?/, '$1TABLE_NAME$2')
+    .replace(/(\/schema\/.*\/views\/)[^/]*(\/.*)?/, '$1VIEW_NAME$2')
+    .replace(/(\/schema\/.*\/functions\/)[^/]*(\/.*)?/, '$1FUNCTION_NAME$2');
 };
 
-const apiExplorerHandler = () => {
-  return API_EXPLORER_PATH;
+const apiExplorerHandler = (path: string) => {
```

Comment:
This is just an identity function, right? Can be removed?

Reply:
yeah. makes sense